reference-artifacts/Custom-Scripts/SEA-uninstall/README.md

```diff
@@ -72,7 +72,10 @@ The logic of the script is the following:
 
 3. Before running this script you must manually delete AWS SSO.
 
-4. Execute the script `python3 aws-sea-cleanup.py`
+4. Execute the script `python3 aws-sea-cleanup.py`.
+
+**Note: ** if you used a different AcceleratorName you can use `python3 aws-sea-cleanup.py --AcceleratorName YOUR_ACCELERATOR_NAME`.
```

Comment:
This should be AcceleratorPrefix (less the -), not Accelerator Name, for consistency :-).